packages/test/test-service-load/src/runner.ts

```diff
@@ -377,17 +377,23 @@ async function setupOpsMetrics(container: IContainer, logger: ITelemetryLogger,
         }
     };
 
-    let submitedOps = 0;
+    let submittedOpsSize = 0;
+    let submittedOps = 0;
     container.deltaManager.on("submitOp", (message) => {
         if (message?.type === "op") {
-            submitedOps++;
+            submittedOps++;
+            var currOpSize = (JSON.stringify(message)).length;
+            submittedOpsSize += currOpSize;
         }
     });
 
+    let receivedOpsSize = 0;
     let receivedOps = 0;
     container.deltaManager.on("op", (message) => {
         if (message?.type === "op") {
             receivedOps++;
+            var currOpSize = (JSON.stringify(message)).length;
+            receivedOpsSize += currOpSize;
```

Comment:
i don't think we should get the size of every op. also, this is a sum of op sizes, but you don't log the count at the same time.

Reply:
i also think this will be highly variable based on which ops. what is the goal here? what are we trying to learn or achieve?

Reply:
Hey Anthony, there are a few changes which should have been merged after which this PR would make much more sense in terms of the usage. Let me get those PRs created and then we can come back to this PR. Thanks!!